### PR TITLE
Enable coveralls

### PR DESCRIPTION
**Pull request by karelhala**
Tests are running, however there was no coverage report tool, so enable [coveralls.io](https://coveralls.io/github/ManageIQ/ui-components)

[karma coverage](https://karma-runner.github.io/0.8/config/coverage.html) is used for generating coverage data.

Current code coverage:
selection_073

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -27,14 +27,15 @@ module.exports = function(config) {
...
 
     // web server port
     port: 9876,
 
     preprocessors: {
-      [fileGlob]: ['webpack'],
-      [jsonGlob]: ['webpack']
+      [fileGlob]: ['webpack', 'coverage'],
+      [jsonGlob]: ['webpack'],
+      applicationFile: 'coverage'
```
Comment (himdel) [inline karma.conf.js]: [applicationFile]?

Comment (karelhala) [inline karma.conf.js]: No need to put it in array, but to be consistent over the file, I'll update it.

Comment (himdel) [inline karma.conf.js]: Wasn't talking about the array, but you fixed both, thanks :)

**Comment by himdel** [comment]
Merged, fingers crossed :)

**Pull request merged**